kinsumer.go

```diff
@@ -388,3 +424,51 @@ func (k *Kinsumer) Next() (data []byte, err error) {
 
 	return data, err
 }
+
+// CreateRequiredTables will create the required dynamodb tables
+// based on the applicationName
+func (k *Kinsumer) CreateRequiredTables() error {
```

Comment:
Thoughts on running these in go routines with errgroup? That would build the tables faster.